packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -34,10 +34,19 @@ import FormSubHeader from 'Components/form-sub-header';
 import FormSubmitErrorMessage from 'Components/form-submit-error-message';
 import LoadErrorMessage from 'Components/load-error-message';
 
-const removeObjProperties = (property_arr, { ...obj }) => {
-    property_arr.forEach((property) => delete obj[property]);
-    return obj;
-};
+const removeObjProperties = (property_arr, { ...obj }) =>
```

Comment:
The previous `removeObjProperties` implementation seems easier, what's the reason for changing it?